components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -425,6 +433,25 @@ - (void)setDrawerShadowColor:(UIColor *)drawerShadowColor {
   self.shadowedView.shadowLayer.shadowColor = drawerShadowColor.CGColor;
 }
 
+- (void)setScrimShouldAdoptTrackingScrollViewBackgroundColor:
+    (BOOL)scrimShouldAdoptTrackingScrollViewBackgroundColor {
+  if (_scrimShouldAdoptTrackingScrollViewBackgroundColor !=
+      scrimShouldAdoptTrackingScrollViewBackgroundColor) {
+    _scrimShouldAdoptTrackingScrollViewBackgroundColor =
+        scrimShouldAdoptTrackingScrollViewBackgroundColor;
+    if ([self.delegate respondsToSelector:@selector
+                       (bottomDrawerContainerViewControllerNeedsScrimAppearanceUpdate:
+                                    scrimShouldAdoptTrackingScrollViewBackgroundColor:)]) {
+      [self.delegate bottomDrawerContainerViewControllerNeedsScrimAppearanceUpdate:self
+                                 scrimShouldAdoptTrackingScrollViewBackgroundColor:
+                                     _scrimShouldAdoptTrackingScrollViewBackgroundColor];
+    }
+  }
+  self.shadowedView.layer.shadowColor = _scrimShouldAdoptTrackingScrollViewBackgroundColor
```

Comment:
lets mention the bug in a comment above this line here too.
    // This logic is to mitigate b/119714330.